### ISSUE TITLE
did anyone test this code? bunch of missing files. 

### ISSUE THREAD

**Issue by johndpope** [PacktPublishing/Machine-Learning-with-Core-ML#4]
<img width="1104" alt="screen shot 2018-08-16 at 12 06 18 pm" src="https://user-images.githubusercontent.com/289994/44220532-d81a7880-a14c-11e8-9106-cd54e32de746.png">
<img width="266" alt="screen shot 2018-08-16 at 12 07 14 pm" src="https://user-images.githubusercontent.com/289994/44220551-ecf70c00-a14c-11e8-9d33-b90d2cc72757.png">

your projects are hard coded to your Joshua.Newnham/Dropbox/

you should be using git to push / pull files not dropbox.

**Comment by johndpope** [comment]
I think folder structure is easier. 
fyi - 
https://www.amazon.com/dp/1787121518#customerReviews
<img width="636" alt="screen shot 2018-08-16 at 12 08 26 pm" src="https://user-images.githubusercontent.com/289994/44220615-17e16000-a14d-11e8-8fa6-839d592a9435.png">

**Comment by joshnewnham** [comment]
@johndpope Thanks for highlighting this - I'll look into updating the paths and update the repo.

**Comment by joshnewnham** [comment]
Hi @johndpope - just to let you know that I have updated the references and apologies for the inconvenience. Regarding your question 'Did anyone test this code?' - yes; there are technical reviewers who tested the code but possibly didn't see this as a concern or priority. I will add that I had thought I had addressed this previously but obviously something went wrong during merging. Thanks again for your message - no doubt save many others having to spend time re-adding the files.
